cmd/podman/root.go

```diff
@@ -62,7 +62,13 @@ Options:
 
 var (
 	rootCmd = &cobra.Command{
-		Use:                   filepath.Base(os.Args[0]) + " [options]",
+		// In shell completion, there is `.exe` suffix on Windows.
+		// This does not provide the same experience across platforms
+		// and was mentioned in [#16499](https://github.com/containers/podman/issues/16499).
+		//
+		// It was waiting for a fix in cobra upstream, but since that has not happened yet,
+		// a fix was implemented in [#24966](https://github.com/containers/podman/pull/24966).
```

Comment:
I don't think that this is necessary and anyway the cobra upstream issue [has been fixed](https://github.com/spf13/cobra/pull/1960).
```suggestion
```